pkg/workloads/console/controller.go

```diff
@@ -166,15 +168,21 @@ func (r *reconciler) Reconcile() (res k8rec.Result, err error) {
 	}
 
 	// Requeue reconciliation if the status may change
-	switch r.console.Status.Phase {
-	case workloadsv1alpha1.ConsolePending:
+	switch {
+	case r.console.Pending():
 		res = requeueAfterInterval(r.logger, time.Second)
-	case workloadsv1alpha1.ConsoleRunning:
+	case r.console.Running():
 		res = requeueForExpiration(r.logger, r.console.Status)
+	case r.console.Stopped():
+		// Requeue for when the console needs to be deleted
+		// In the future we could allow this to be configured in the template
+		res = requeueAfterInterval(r.logger, ConsoleTTL)
```

Comment:
You may want to undo this in the conditional block below. Otherwise we're going to also try to reconcile this 24h *after* the console is deleted

Reply:
True - although that's kinda harmless since we'll bail when we can't find the console.

Reply:
Yeah there's no harm, but best to avoid useless/confusing noise in the logs if we can I reckon